### PR TITLE
Bloodname GM Tools

### PR DESCRIPTION
**Pull request by Windchild292**
This removes the Random Bloodname Dialog and moves it to be part of the GM Tools Dialog.

As part of this I standardized the names between Faction and Clan.

### CODE REVIEW

File: MekHQ/src/mekhq/gui/dialog/GMToolsDialog.java [hunk 1]
```diff
@@ -84,28 +90,58 @@
...
+    //region Data Sources
+    private CampaignGUI gui;
+    private Person person;
+    private List<Clan> clans = new ArrayList<>();
+    //endregion Data Sources
+
+    //region Bloodname Information
+    int originClan;
+    Phenotype selectedPhenotype;
+    int bloodnameYear;
```
Comment (sixlettervariables) [inline MekHQ/src/mekhq/gui/dialog/GMToolsDialog.java:134]:
```suggestion
    private int originClan;
    private Phenotype selectedPhenotype;
    private int bloodnameYear;
```

File: MekHQ/src/mekhq/gui/dialog/GMToolsDialog.java [hunk 2]
```diff
@@ -413,6 +451,151 @@ private JPanel getCallsignGenerator() {
...
+
+                setEnabled(list.isEnabled());
+
+                if (cellHasFocus) {
+                    Border border = null;
+                    if (isSelected) {
+                        border = DefaultLookup.getBorder(this, ui, "List.focusSelectedCellHighlightBorder");
+                    }
+                    if (border == null) {
+                        border = DefaultLookup.getBorder(this, ui, "List.focusCellHighlightBorder");
```
Comment (sixlettervariables) [inline MekHQ/src/mekhq/gui/dialog/GMToolsDialog.java:525]: I get errors when building with `DefaultLookup.*` under JDK11. Perhaps this should move to [`UIManager`](https://docs.oracle.com/javase/8/docs/api/javax/swing/UIManager.html)?

**Review by sixlettervariables — COMMENTED** [review]
One comment regarding DefaultLookup.

File: MekHQ/src/mekhq/gui/dialog/GMToolsDialog.java (hunk 2 above)
Comment (Windchild292) [reply MekHQ/src/mekhq/gui/dialog/GMToolsDialog.java]: Fixed by reworking how the override works

**Review by sixlettervariables — APPROVED** [review]

**Pull request merged**